### PR TITLE
Adding a way to send native analytics from the micro apps

### PR DESCRIPTION
**Pull request by dmarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Messages are sent from the webview with a specific event type that the native ap
 | `share.text` | µApp | Launch the native sharing components to be able to copy and paste or send the provide text. |
 | `session.token` | µApp | Generate a JSON Web Token, with no additional payload, for the current microapp suitable for authenticating the user from an external service with the OAuth Applications secret client key. |
 | `session.sign` | µApp | Like for `session.token`, will generate a JSON Web Signature for the current microapp, but containing additional signed data provided in the params. *NOTE:* In the future, this request *may* cause a popup to be launched for the user's permission. |
+| `analytics.event` | µApp | Sends event using the native SDK. |
```
Comment (MarkVillacampa) [inline README.md]: We need to document the parameters it accepts. To be consistent with the native apps I'd go with `name` (String) and `properties` (Object).

You can check the way other methods are documented and add this one.

Comment (dmarin) [reply README.md]: actually that was one of my questions :). I guess we will expect some properties as the event title. Are we validating the payloads? I did not see in the readme examples for every "topic", do you think is worth adding them?

Comment (MarkVillacampa) [reply README.md]: We only document the methods that either accept parameters or return payloads. The payload is validated on the native side when deserializing it into the `Message` object.

**Review by MarkVillacampa — APPROVED** [review]
Looks good!

File: src/AriesSDK.js [hunk 2]
```diff
@@ -65,6 +65,10 @@ sdk.scanIdentity = function(params) {
 	return invoke('scan.identity', params)
 }
 
+sdk.analyticsEvent = function(params) {
+	return invoke('analytics.event', parans)
```
Comment (samlown) [inline src/AriesSDK.js:69]: Typo!

Comment (dmarin) [reply src/AriesSDK.js:69]: nice catch! though this is already merged, what do you guys usually do in these situations? commit straight to master or a new pull request fixing the typo?

Comment (MarkVillacampa) [reply src/AriesSDK.js:69]: a new version hasn't been tagged  or released so just open a new PR and merge with master.

**Pull request merged**